delphi/app/routes.py

```diff
@@ -1,23 +1,17 @@
 from app import app
 from flask import request
-import cv2
-import numpy
 
 
-@app.route('/')
-@app.route('/index')
-def index():
-    return 'Hello, World!'
+@app.route('/update-sensor-data', methods=['POST'])
+def updateSensorData():
+    data = request.get_json(silent=True)
 
+    if data is not None:
```

Comment:
In python, if you do `if data`, then it will automatically check if it's null or not (true => objc isn't null, false => object is null)

Reply:
If data is a boolean, than doing `if data` would not work. Has `is not None` conveys more meaning.